compiler/rustc_parse/src/errors.rs

```diff
@@ -348,6 +350,13 @@ pub(crate) enum IfExpressionMissingThenBlockSub {
     AddThenBlock(#[primary_span] Span),
 }
 
+#[derive(Subdiagnostic)]
+#[help(parse_extra_if_in_let_else)]
+pub(crate) struct IfExpressionLetSomeSub {
+    #[primary_span]
+    pub if_span: Span,
+}
```

Comment:
This should have been a structured suggestion, not a help, so that rustfix can apply the change. The missing block help isn't a structured suggestion because we didn't have any code for it (but arguably it should have been as well).

Reply:
Is this just changing the `help` to a `suggestion`? If so I can address this in a follow-up PR.

Reply:
Yup